src/Message/Payload/LtiMessagePayload.php

```diff
@@ -218,6 +218,15 @@ public function getProctoringVerifiedUser(): ?ProctoringVerifiedUserClaim
         return $this->getClaim(ProctoringVerifiedUserClaim::class);
     }
 
+    public function getProctoringEndAssessmentReturn(): bool
+    {
+        if ($this->hasClaim(static::CLAIM_LTI_PROCTORING_END_ASSESSMENT_RETURN)) {
+            return boolval($this->getClaim(static::CLAIM_LTI_PROCTORING_END_ASSESSMENT_RETURN));
```

Comment:
To be honest, I don't see a single reason to continue using a function invocation when we have type-casting.
```suggestion
            return (bool)$this->getClaim(static::CLAIM_LTI_PROCTORING_END_ASSESSMENT_RETURN);
```

Reply:
for consistency with other casts in this class (intval), would you prefer if I refactor them all ? :)

Reply:
If that's not too much of a trouble ;)

Reply:
no worries :) done